src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = props => {
+  const {
+    id, title, category,
+  } = props;
```

Comment:
**OPTIONAL**: I saw you are destructuring on the argument on `BooksList`, you can keep the same pattern to be consistent, just remember that the `book` should be a `prop`, not the properties of the book.

```js
const Book = ({ book }) => {
```

Reply:
Done :D